src/pages/billing/List/List.ts

```diff
@@ -137,7 +140,7 @@ export const List = (props: ListProps) => {
 
   // Render
   const breadcrumbs = 'Billing > Billing Project'
-  const billingProjectListWidth = 330
+  const billingProjectListWidth = 350
```

Comment:
Increased the sidebar with to take into account 16px now being used for the delete icon. There's nothing on the right-hand side that is particularly sensitive to the width of the entire browser.